### DISCUSSION TITLE
What is frigate+

### DISCUSSION THREAD

**Discussion by husnainali09** [blakeblackshear/frigate#3649] [General]
I really dont understand what it does or how to incorporate it into the beta 11v im running. Could someone fill me in ?

**Comment by NickM-27** [comment]
The current model that is used in frigate is the google coral test data model. While it has been instrumental in frigate thus far, it is not very accurate for certain objects and wasn't trained on images from security cameras. 

Frigate+ allows the submission of images from users which will be used to train new models that can be used and will be much more accurate. It's also on the roadmap to allow users to create labels and submit pictures of things that they want to track that doesn't fit for others

In 0.11 you can upload those pictures to frigate+ with 1 click where they will then be annotatable

**Reply by husnainali09** [reply]
Ah. So frigate will be able tell my Labrador from my staffy?

**Reply by NickM-27** [reply]
I don't believe that is really the intended use-case, but I suppose it is possible? It's mainly, for example, you wanted to track bears but naturally `bear` isn't useful for someone that lives in the city.

Frigate+ will include in the default model delivery vehicles like UPS, Fedex, and Amazon which is another example

**Reply by husnainali09** [reply]
My current use of frigate is security and presence detection along with PIR ( say Mrs is cooking and lights don't return off as PIR might not detect motion). Being able to tell me and Mrs also will open up bunch of automation options

**Reply by NickM-27** [reply]
That is already possible with https://github.com/jakowenko/double-take which integrates directly with frigate
